ScoutSuite/providers/aws/resources/vpcs.py

```diff
@@ -12,16 +13,25 @@ def __init__(self, facade, scope: dict, add_ec2_classic=False):
         self.add_ec2_classic = add_ec2_classic
 
     async def fetch_all(self, **kwargs):
-        raw_vpcs = await self.facade.ec2.get_vpcs(self.scope['region'])
-        for raw_vpc in raw_vpcs:
-            vpc_id, vpc = self._parse_vpc(raw_vpc)
-            self[vpc_id] = vpc
 
-        await self._fetch_children_of_all_resources(
-            resources=self,
-            scopes={vpc_id: {'region': self.scope['region'], 'vpc': vpc_id}
-                    for vpc_id in self}
-        )
+        try:
```

Comment:
Exception handling is already done in EC2 facade (thanks to your modifications above), what's the point of doing it again here?